test/io/configs/expected/no-defaults-with-db-other-authenticator.config

```diff
@@ -19,6 +19,7 @@ jwt-secret = "ODERREALLYREALLYREALLYREALLYVERYSAFE"
 jwt-secret-is-base64 = true
 log-level = "info"
 openapi-mode = "disabled"
+openapi-security-active = true
```

Comment:
Hm, I think this one required changing the in-db config(like below). Would be good to test this too.

https://github.com/PostgREST/postgrest/blob/bdf1cbe11126bae043b754009a1a7fe19f9c7e9f/test/io/db_config.sql#L56-L57